### PR TITLE
Add simplified view for "tsuru app info"

### PR DESCRIPTION
**Pull request by wpjunior**
Example of output:

tsuru app info -a myapp -s

```
Application: myapp
Platform: python
Plan: c1m1
Pool: mypool (docker)
Teams: team (owner), ops
Cluster External Addresses: myapp.globo.com (cname), routeraddr.router.com
Bound Services: service/instance1, service/instance2
Units: 4
+---------+-------+
| Process | Units |
+---------+-------+
| web     | 4     |
+---------+-------+
```

### CODE REVIEW

File: tsuru/client/apps.go [hunk 1]
```diff
@@ -590,24 +629,53 @@ func ShortID(id string) string {
...
 {{ end -}}
 Application: {{.Name}}
-Description:{{if .Description}} {{.Description}}{{end}}
-Tags:{{if .TagList}} {{.TagList}}{{end}}
+{{- if .Description }}
+Description: {{.Description}}
+{{- end }}
+{{- if .TagList }}
+Tags: {{.TagList}}
+Created by: {{.Owner}}
```
Comment (tcarreira) [inline tsuru/client/apps.go:641]: This `Created by: {{.Owner}}` is inside the `{{- if .TagList }}`

File: tsuru/client/apps.go [hunk 2]
```diff
@@ -590,24 +629,53 @@ func ShortID(id string) string {
...
+{{- end }}
+Platform: {{.Platform}}
+Plan: {{ .Plan.Name }}
+Pool: {{.Pool}} ({{ .Provisioner }}{{ if .Cluster}} cluster: {{ .Cluster }}{{end}})
```
Comment (tcarreira) [inline tsuru/client/apps.go:645]: What do you think about adding a separator between pool and cluster?
Maybe 
```
Pool: {{.Pool}} ({{ .Provisioner }}{{ if .Cluster}} | cluster: {{ .Cluster }}{{end}})
```

File: tsuru/client/apps.go [hunk 3]
```diff
@@ -590,24 +629,53 @@ func ShortID(id string) string {
...
+{{if not .Routers -}}
```
Comment (tcarreira) [inline tsuru/client/apps.go:646]: If .Routers exist, shouldn't if be listed also?

File: tsuru/client/apps.go [hunk 4]
```diff
@@ -590,24 +629,53 @@ func ShortID(id string) string {
...
+Router:{{if .Router}} {{.Router}}{{if .RouterOpts}} ({{.GetRouterOpts}}){{end}}{{end}}
+{{end -}}
+Teams: {{.TeamList}}
+{{- if .InternalAddr }}
```
Comment (tcarreira) [inline tsuru/client/apps.go:650]: I thought `.InternalAddr` always exists, but `.ExternalAddr` may exist or not.
If that's true, there is a missing `{{- if .ExternalAddr }}`

File: tsuru/client/apps.go [hunk 5]
```diff
@@ -590,24 +629,53 @@ func ShortID(id string) string {
...
+{{- if .SimpleServicesView }}
+Bound Services: {{ .SimpleServicesView }}
+{{- end }}
+`
+
+const fullFormat = `{{ if .Error -}}
+Error: {{ .Error }}
+{{ end -}}
+Application: {{.Name}}
+{{- if .Description }}
```
Comment (tcarreira) [inline tsuru/client/apps.go:663]: I would keep the old behavior in this one (showing empty description), as this is the fullFormat.
I find it more explicit than omitting. 

What do you think? 
(the same for the rest of the template)

Comment (tcarreira) [reply tsuru/client/apps.go:663]: Also, this change forces rewriting some tests, which is considered breaking the contract for some (like myself, sometimes 😄 )

File: tsuru/client/apps_test.go [hunk 6]
```diff
@@ -2333,12 +2293,12 @@ func (s *S) TestAppListUnitWithoutID(c *check.C) {
...
-	expected := `+-------------+-----------+---------------+
-| Application | Units     | Address       |
-+-------------+-----------+---------------+
-| app1        | 1 started | app1.tsuru.io |
-|             |           | 10.10.10.10   |
-+-------------+-----------+---------------+
+	expected := `+-------------+-----------+-----------------------+
+| Application | Units     | Address               |
++-------------+-----------+-----------------------+
+| app1        | 1 started | app1.tsuru.io (cname) |
```
Comment (tcarreira) [inline tsuru/client/apps_test.go:2299]: I like this "breaking change" though 😉

**Review by tcarreira — CHANGES_REQUESTED** [review]
I loved your work. Very nicely done.
I made some suggestions nevertheless (some are opinions, some are minor bugs)

File: tsuru/client/apps.go (hunk 3 above)
Comment (wpjunior) [reply tsuru/client/apps.go:646]: .Router is a legacy method from old tsuru apis, @tcarreira

**Pull request merged**